src/AppBundle/Controller/Security/AdherentSecurityController.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+namespace AppBundle\Controller\Security;
+
+use AppBundle\Form\LoginType;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Method;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Route;
+use Symfony\Bundle\FrameworkBundle\Controller\Controller;
+
+/**
+ * @Route("/espace-adherent/connexion")
+ */
+class AdherentSecurityController extends Controller
+{
+    /**
+     * @Route(name="adherent_login")
+     * @Method("GET")
+     */
+    public function loginAction()
+    {
+        $securityUtils = $this->get('security.authentication_utils');
+
+        $form = $this->createForm(LoginType::class, ['_adherent_email' => $securityUtils->getLastUsername()], [
```

Comment:
then how do you do a "Configured security for members login" part without a successfull login ? 🤔